bundles/org.openhab.binding.revogismartstripcontrol/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.2-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>2.5.3-SNAPSHOT</version>
```

Reply:
Your PR build is failing because this change was never committed.

Reply:
I update that version again.
It's a bit weird, I got an email from travis about the fixed build, but looking into travis shows still failures:

```
[ERROR]     Non-resolvable parent POM for org.openhab.addons.bundles:org.openhab.binding.revogismartstripcontrol:2.5.3-SNAPSHOT: Failure to find org.openhab.addons.bundles:org.openhab.addons.reactor.bundles:pom:2.5.3-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of sonatype-snapshots has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 6, column 11 -> [Help 2]
```